themes/minimal/layouts/index.html

```diff
@@ -18,7 +18,27 @@ <h2>{{ .Site.Title }}</h2>
         <h3>{{ markdownify .Site.Params.Description }}</h3>
         {{ with .Content }} <div class="homepage-content text-justify">{{ . }}</div> {{ end }}
 
-        <img src="images/sars-cov-2-600x300.jpg">
+        <img style="width:60%;" src="images/sars-cov-2-600x300.jpg">
+
+        <h3> What is this hub about? </h3>
+        <p align="justify" style="font-size:120%;">
+            This site provides a community-driven data repository and curation service for molecular structures, models,
+            therapeutics, and simulations related to computational research on the SARS-CoV-2 / COVID-19 / "Coronavirus"
+	        pandemic. For more info about this project, see the <a href="/about">about</a> page. We gladly accept any of
```

Comment:
"We gladly accept any" data is going to be problematic, since contributors may not be happy if their submissions are reviewed and potentially rejected. Instead, could we be clear about how people can contribute and how it will be reviewed by linking to the contributions page, while still welcoming contributions through this process?